cmd/containerboot/settings.go

```diff
@@ -105,8 +102,9 @@ func configFromEnv() (*settings, error) {
 		PodIP:                                 defaultEnv("POD_IP", ""),
 		EnableForwardingOptimizations:         defaultBool("TS_EXPERIMENTAL_ENABLE_FORWARDING_OPTIMIZATIONS", false),
 		HealthCheckAddrPort:                   defaultEnv("TS_HEALTHCHECK_ADDR_PORT", ""),
-		LocalAddrPort:                         defaultEnv("TS_LOCAL_ADDR_PORT", defaultLocalAddrPort),
+		LocalAddrPort:                         defaultEnv("TS_LOCAL_ADDR_PORT", "[::]:9002"),
 		MetricsEnabled:                        defaultBool("TS_METRICS_ENABLED", false),
+		HealthEnabled:                         defaultBool("TS_HEALTH_ENABLED", false),
```

Comment:
```suggestion
		HealthCheckEnabled:                         defaultBool("TS_HEALTHCHECK_ENABLED", false),
```

Reply:
We should also add that env var to the list here https://github.com/tailscale/tailscale/blob/13faa64c142148b1f8c8afd22d61e4a0de651b98/cmd/containerboot/main.go#L14

Reply:
How would you feel about `TS_HEALTH_CHECK_ENABLED` instead?

Reply:
@tomhjp actually thinking about this more, the names will be inconsistent with our other env vars/CLI commands which are all in imperative form `TS_ACCEPT_DNS`, `--accept-routes` etc (not `TS_DNS_ENABLED`)
wdyt maybe we could make these `TS_ENABLE_METRICS/HEALTH_CHECK`?